core-bundle/src/EventListener/RequestTokenListener.php

```diff
@@ -54,15 +54,15 @@ public function __invoke(RequestEvent $event): void
 
         // Only check the request token if a) the request is a POST request, b)
         // the request is not an Ajax request, c) the _token_check attribute is
-        // not false, d) the _token_check attribute is set or the request is a
+        // not false, d) the _token_check attribute is true or the request is a
         // Contao request and e) the request has cookies, an authenticated user
         // or the session has been started
         if (
             'POST' !== $request->getRealMethod()
             || $request->isXmlHttpRequest()
             || false === $request->attributes->get('_token_check')
```

Comment:
```suggestion
            || ($request->attributes->has('_token_check') && !$request->attributes->get('_token_check'))
```
Should we change this as well then so that a falsey `_token_check` has the same meaning for Contao routes too?

Reply:
I would prefer to only support true and false.